<!-- saved from url=(0047)http://test.galaxycloud.org/static/welcome.html -->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

  <head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <link rel="stylesheet" href="style/base.css" type="text/css">
    <link rel="stylesheet" href="./eupath-galaxy.css" type="text/css">
  </head>

  <body style="margin: 0!important; padding: 0;">
    <div class="document">

      <!-- HEADER -->
      <div class="eupath-header" align="center">
        <table width="95%">
          <tbody><tr>
            <!--<td><img class="topLogo" src="http://eupathdb.org/eupathdb/images/EuPathDB/title_s.png"></td>-->
            <td><img class="topLogo" src="images/VEuPathDB.png"></td>
            <td class="logo_table_row">GG-v5.4</td>
            <!--<td class="status"><div class="status_image" alt="Globus Genomics System Status"></div></td>-->
          </tr>
          <tr>
            <td colspan=2 style="font-size:70%">
              <i><span style="color:red;font-weight:600">News:</span>  This is a new version of Galaxy with updated tools and features. Please <a href="https://veupathdb.org/veupathdb/app/contact-us" target="_top">Contact Us</a> for help or if there are any problems.</i>
            </td>
          </tr>
 	  <tr>
            <td colspan=2 style="font-size:70%">
              <i><span style="color:red;font-weight:600">Attention:</span> We periodically purge your inactive datasets (inputs and outputs). Any dataset that has not been updated in the past 60 days will be removed. Please backup/download your datasets to keep long term, and delete unwanted histories. Thank you for your cooperation.</i>
            </td>
	  </tr><tr></tr>
          </tbody>
        </table>
      </div>

      <!-- TITLE -->
      <h1 id="eupath-title"></h1>
      <div id="eupath-subtitle"></div>

      <!-- INTRO (help section)  -->
      <div class="eupath-section">
        <h4>With VEuPathDB Galaxy you can:</h4>
        <div id="eupath-intro">
          (Do nothing currently)
        </div>
      </div>
      <br>

      <!-- WORKFOW LIST -->
      <div class="eupath-section">
        <h2>Get started with VEuPathDB pre-configured workflows:</h2>
        <div>
        <h3>OrthoMCL</h3>
	<p style="margin-left: 40px">
	This workflow uses BLASTP and the OrthoMCL algorithm to assign your set of proteins to OrthoMCL groups. Usually, the most recent version (e.g., OG6r5) is in sync with the groups on the <a href="https://orthomcl.org" target="_blank">OrthoMCL website</a>, but you can also select a previous set such as OG5. Explore this <a href='https://static-content.veupathdb.org/documents/ProteinsToOrthoMCL.pdf' target='_blank'>OrthoMCL workflow tutorial</a> to learn more.
	<br> </p>
	<ul><ul>
	  <li>
	    <a id='workflow8' href='javascript:void(0)' onClick="import_and_run_workflow('4abff78227b71790')">Workflow to map your proteins to OrthoMCL groups</a>
	  </li>
	</ul></ul>

	<h3>RNA-seq</h3>

	<p style="margin-left: 40px">

	Use the following workflows to analyze your FASTQ files. The workflows use FASTQ groomer and Trimmomatic for preparation of reads, FASTQC for sequencing statistics, and HISAT2 for mapping reads to a VEuPathDB reference genome. Choose the appropriate workflow based on your input data and your desired analysis. Explore this <a href='https://static-content.veupathdb.org/documents/VEuPathDB_RNAseq_export.pdf' target='_blank'>RNA-Seq export tutorial</a> to learn about exporting your workflow results to VEuPathDB.
	<br><br>

	<b>Examine genome coverage and calculate TPM for each gene</b>
	<br>
	In addition to the tools described above, these workflows use three tools (bamCoverage, htseq-count, HTSeqCountToTPM) to generate BigWig and TPM files that can be analyzed on VEuPathDB, in Galaxy, or on your computer. The workflows take any number of samples and processes them in parallel. To export the results to VEuPathDB, use the 'RNA-Seq to VEuPathDB' tool.
	</p>
	<ul><ul>
	  <li>
	    <a id='workflow9' href='javascript:void(0)' onClick="import_and_run_workflow('173604cba3d7390d')">Workflow for paired-end stranded reads</a>
	  </li>
	  <li>
	    <a id='workflow9' href='javascript:void(0)' onClick="import_and_run_workflow('24f0a125c6a4790e')">Workflow for paired-end unstranded reads</a>
	  </li>
	  <li>
	    <a id='workflow9' href='javascript:void(0)' onClick="import_and_run_workflow('4f09bdb82cce614a')">Workflow for single-end stranded reads</a>
	  </li>
	  <li>
	    <a id='workflow9' href='javascript:void(0)' onClick="import_and_run_workflow('0ae16e632d559c8d')">Workflow for single-end unstranded reads</a>
	  </li>
	</ul></ul>

	<p style="margin-left: 40px">
	<b>Identify genes with statistically significant expression differences between two samples</b>
	<br>
	In addition to the tools described above, these workflows use three tools (htseq-count, DESeq2, Bam to BigWig) to determine whether each gene exhibits differential expression and to generate BigWig coverage files. The output files can be analyzed in Galaxy or on your computer. The workflows compare two samples with any number of replicates. To export your BigWig files to VEuPathDB, use the 'Bigwig Files to VEuPathDB' tool. To filter your DESeq2 result file and obtain a set of Gene IDs that change significantly (defaults: fold-change>=2 and adj-p<=0.05; these can be changed), use <a id='workflow9' href='javascript:void(0)' onClick="import_and_run_workflow('fa015f4a720f3a0d')">this workflow</a>. Copy and paste the Gene IDs into the 'Identify Genes based on Gene ID(s)' question on a VEuPathDB website, <a href="https://beta.plasmodb.org/plasmo.beta/app/search/transcript/GeneByLocusTag" target="_blank">as seen here for the PlasmoDB site</a>.
	</p>
	<ul><ul>
	  <li>
            <a id='workflow9' href='javascript:void(0)' onClick="import_and_run_workflow('dffc100640c11d9f')">Workflow for paired-end stranded reads</a>
          </li>
          <li>
            <a id='workflow9' href='javascript:void(0)' onClick="import_and_run_workflow('76606d4e528e4a6c')">Workflow for paired-end unstranded reads</a>
          </li>
          <li>
            <a id='workflow9' href='javascript:void(0)' onClick="import_and_run_workflow('a0f697795c83a1ee')">Workflow for single-end stranded reads</a>
          </li>
          <li>
            <a id='workflow9' href='javascript:void(0)' onClick="import_and_run_workflow('75a9f6c1713aeea5')">Workflow for single-end unstranded reads</a>
	  </li>
	</ul></ul>

	    <h3>Variant calling</h3>

<p style="margin-left: 40px">
	    Use the following workflows to analyze your FASTQ files. The workflows use Sickle for preparation of reads, Bowtie2 for mapping reads to a VEuPathDB reference genome, Freebayes for variant detection, SnpEff to evaluate the effect of variants, and SnpSift for filtering types of variants. Choose the appropriate workflow based on your input data. A VCF file is generated that can be analyzed in Galaxy or downloaded to your computer. NOTE: Export of VCF files to VEuPathDB will be available soon.
	    <br>
</p>
	    <ul><ul>
	      <li>
		<a id='workflow9' href='javascript:void(0)' onClick="import_and_run_workflow('82c0ae1b3244c64c')">Workflow for single-end reads</a>
	      </li>
	      <li>
		<a id='workflow9' href='javascript:void(0)' onClick="import_and_run_workflow('6b19ec4fe66be0fd')">Workflow for paired-end reads</a>
	      </li>
	      </ul></ul>

            <h3>Additional VEuPathDB workflows</h3>

<p style="margin-left: 40px">
  <a href="https://veupathdbprod.globusgenomics.org/workflow/list_published" target="_blank">Additional VEuPathDB workflows</a> can be accessed in the menu above: 'Shared Data' >> 'Workflows'.
            <br>
</p>

        </div> 
      </div>
    </div>  <!-- end document -->

    <!-- FOOTER -->
    <hr style="margin-bottom:0">

    <p style="font-size:70%;background-color:white;padding:5px 5px 0;text-align:left">
VEuPathDB Galaxy workspaces are provided free of charge. <b>Note that data will be purged after 60 days of inactivity.</b> We encrypt data transfers and storage but ultimately, we cannot guarantee the security of data transmissions between VEuPathDB, Globus and affiliates, Amazon Cloud Services, and the user. It is your responsibility to back up your data and obtain any required permissions from your study and/or institution prior to uploading data for analyses on the VEuPathDB Galaxy platform. Do not use, transmit, upload or share any human identifiable information in the files you analyze. VEuPathDB, the multiple universities responsible for providing this resource, Globus and affiliates, and Amazon Cloud Services are not responsible or liable for the loss and/or release of any data you analyze via the VEuPathDB Galaxy platform.
<a href="https://veupathdb.org/veupathdb/app/static-content/privacyPolicy.html" target="_blank">See the VEuPathDB privacy policy.</a></p>
   
    <footer class="eupath-footer">
      <div>

      <div class="row">
        <div class="col-md-12">
          <ul class="list-inline">
            <li title="VEuPathDB.org">
              <a href="http://veupathdb.org" target="_top">
                <img width="25" src="http://eupathdb.org/oauth/assets/images/veupath-icon.png"/></a></li>
            <li title="AmoebaDB.org">
              <a href="http://amoebadb.org" target="_top">
                <img src="http://eupathdb.org/eupathdb/images/AmoebaDB/footer-logo.png"/></a></li>
            <li title="CryptoDB.org">
              <a href="http://cryptodb.org" target="_top">
                <img src="http://eupathdb.org/eupathdb/images/CryptoDB/footer-logo.png"/></a></li>
            <li title="FungiDB.org">
              <a href="http://fungidb.org" target="_top">
                <img src="http://eupathdb.org/eupathdb/images/FungiDB/footer-logo.png"/></a></li>
           <li title="GiardiaDB.org">
              <a href="http://giardiadb.org" target="_top">
                <img src="http://eupathdb.org/eupathdb/images/GiardiaDB/footer-logo.png"/></a></li>
           <li title="HostDB.org">
              <a href="http://hostdb.org" target="_top">
                <img width="25" src="http://eupathdb.org/oauth/assets/images/hostdb-icon.png"/></a></li>
            <li title="MicrosporidiaDB.org">
              <a href="http://microsporidiadb.org" target="_top">
                <img src="http://eupathdb.org/eupathdb/images/MicrosporidiaDB/footer-logo.png"/></a></li>
            <li title="PiroplasmaDB.org">
              <a href="http://piroplasmadb.org" target="_top">
                <img src="http://eupathdb.org/eupathdb/images/PiroplasmaDB/footer-logo.png"/></a></li>
            <li title="PlasmoDB.org">
              <a href="http://plasmodb.org" target="_top">
                <img src="http://eupathdb.org/eupathdb/images/PlasmoDB/footer-logo.png"/></a></li>
            <li title="ToxoDB.org">
              <a href="http://toxodb.org" target="_top">
                <img src="http://eupathdb.org/eupathdb/images/ToxoDB/footer-logo.png"/></a></li>
            <li title="TrichDB.org">
              <a href="http://trichdb.org" target="_top">
                <img src="http://eupathdb.org/eupathdb/images/TrichDB/footer-logo.png"/></a></li>
            <li title="TriTrypDB.org">
              <a href="http://tritrypdb.org" target="_top">
                <img src="http://eupathdb.org/eupathdb/images/TriTrypDB/footer-logo.png"/></a></li>
            <li title="VectorBase.org">
              <a href="http://vectorbase.org" target="_top">
                <img width="28" src="http://eupathdb.org/oauth/assets/images/vectorbase-icon.png"/></a></li>
            <li style="font-size:160%;color:grey">|</li>
            <li title="OrthoMCL.org">
              <a href="http://orthomcl.org" target="_top">
                <img width="28" src="http://eupathdb.org/eupathdb/images/OrthoMCL/Ortho-3D-lighter-50.png"/></a></li>
            <li title="ClinEpiDB.org">
              <a href="http://clinepidb.org" target="_top">
                <img width="28" src="http://clinepidb.org/ce/images/symbol-small.png"/></a></li>
            <li title="MicrobiomeDB.org">
              <a href="http://microbiomedb.org" target="_top">
                <img width="28" src="http://eupathdb.org/oauth/assets/images/microbiome-icon.png"/></a></li>
          </ul>
        </div>
      </div>

       <div class="row">
        <div class="col-md-12">
          <a href="https://www.globus.org/genomics" target="_top">
            <span class="globus-name">Powered by globus Genomics</span>
            <span class="globus-icon">
              <img src="https://eupathdb.globusgenomics.org/static/images/galaxyIcon_noText.png"/>
            </span>
          </a>
        </div>
      </div>

      <div class="row">
        <div class="col-md-12">
          Please <a href="https://veupathdb.org/veupathdb/app/contact-us" target="_top">Contact Us</a> with any questions or comments.<br><br>
        </div>
      </div>

      </div>
    </footer>


 <!-- TODO should change from CDN to pull jQuery from WDK site -->
  <script
    src="https://code.jquery.com/jquery-1.12.4.min.js"
    integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ="
    crossorigin="anonymous"></script>

   <script type="text/javascript" src="./eupath-galaxy.js"></script>

  </body>
</html>
